main.py

```diff
@@ -1,3 +1,3 @@
-a =('Hello')
+a =('Hello world')
 
 print(a)
```

Comment:
Aici se putae face intr-o singura linie
```suggestion

print('Hello world'
```